lib/migrations/2_deploy_schemes.ts

```diff
@@ -53,7 +53,7 @@ export const arcJsDeployer = (
     console.log(`Deploying schemes to ${network}`);
 
     const DAOToken = await Utils.requireContract("DAOToken");
-    const gasLimit = await computeMaxGasLimit(web3);
+    const gasLimit = await UtilsInternal.computeMaxGasLimit();
     const gasPrice = 10000000000; // 10 Gwei
```

Comment:
const gasPrice = 10000000000; // 10 Gwei ?
from where this come from ?

Reply:
The scripts doubles that in a few specific cases where it doesn't suffice.

Reply:
fix 10 Gwei is a workaround.
please set the appropriate gas price needed.
You can read that from eth gas station ...(alchemy done that)

Reply:
Yeah, I was thinking of that too.  Would you mind waiting on this until I have Truffle 5 working?  As you know, they have lots of migration changes and I'd like to code this in the context of those changes.  I'm doing the work right now of upgrading to Truffle 5.

Reply:
please open an issue for that.